calico/networking/ip-autodetection.md

```diff
@@ -50,39 +50,105 @@ By default, {{site.prodname}} uses the **first-found** method; the first valid I
+{% tabs %}
+  <label:Operator,active:true>
+<%
+As noted previously, the default autodetection method is **first valid interface found** (first-found). To use a different autodetection method, edit the default [Installation]({{site.baseurl}}/reference/installation/api#operator.tigera.io/v1.Installation) custom resource, specifying the method. Below are examples of the supported autodetection methods:
 
-##### Using environment variables and node resource
+> **Note**: To configure the default autodetection method for IPv6 for any of the below methods, use the field `nodeAddressAutodetectionV6`.
+{: .alert .alert-info}
 
-Because you can configure IP address and subnet using either environment variables or node resource, the following table describes how values are synchronized.
+- **IP or domain name**
 
-| **If this environment variable...** | **Is...**                                             | **Then...**                                                  |
-| ----------------------------------- | ----------------------------------------------------- | ------------------------------------------------------------ |
-| IP/IP6                              | Explicitly set                                        | The specified values are used, and the Node resource is updated. |
-|                                     | Set to autodetect                                     | The requested method is used (first-found, can-reach, interface, skip-interface, kubernetes-internal-ip), and the Node resource is updated. |
-|                                     | Not set, but Node resource has IP/IP6 values          | Node resource value is used.                                 |
-| IP                                  | Not set, and there is no IP value in Node resource    | Autodetects an IPv4 address and subnet, and updates Node resource. |
-| IP6                                 | Not set, and there is a no IP6 value in Node resource | No IP6 routing is performed on the node.                     |
+  A reachable destination (IP address or domain). For example:
 
-### How to
+  ```
+  kind: Installation
+  apiVersion: operator.tigera.io/v1
+  metadata:
+    name: default
+  spec:
+    calicoNetwork:
+      nodeAddressAutodetectionV4:
+        canReach: 8.8.8.8
+  ```
 
-- [Change the autodetection method](#change-the-autodetection-method)
-- [Manually configure IP address and subnet for a node](#manually-configure-ip-address-and-subnet-for-a-node)
+- **Including matching interfaces**
 
-#### Change the autodetection method
+  A regular expression in Golang syntax that includes interfaces that match. For example:
+
+  ```
+  kind: Installation
+  apiVersion: operator.tigera.io/v1
+  metadata:
+    name: default
+  spec:
+    calicoNetwork:
+      nodeAddressAutodetectionV4:
+        interface: eth.*
+  ```
+
+- **Excluding matching interfaces**
+
+  A regular expression in Golang syntax that excludes interfaces that match. For example:
```

Comment:
I think for all of these a slightly more explicit wording would be helpful.

e.g., "Calico will choose an address on each node from an interface that does not match the given regex"